GameFruitBasket/GameFruitBasket/Classes/Library.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace GameFruitBasket.Classes
+{
+    public static class Library
+    {
```

Comment:
Why did you define it as a static class?
Let's imagine that your game became popular and you have to extend current Library functionality
What are you going to do?

It isn't a good case for using static classes. Thy to avoid them